net-dialup/freeradius/freeradius-3.0.17.ebuild

```diff
@@ -21,10 +21,13 @@ SLOT="0"
 
 IUSE="
 	debug firebird iodbc kerberos ldap libressl memcached mysql odbc oracle pam
-	pcap postgres python readline rest samba sqlite ssl
+	pcap postgres python readline rest samba sqlite ssl redis
```

Comment:
I don't really think adding USE=redis is related at all to virtual/mysql. It belongs in a separate commit. So does changing maintainer.

Reply:
I seperate commit and fix redis library dep. thanks.